homeassistant/components/keenetic_ndms2/config_flow.py

```diff
@@ -40,17 +42,16 @@ class KeeneticFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
 
     @staticmethod
     @callback
-    def async_get_options_flow(config_entry):
+    def async_get_options_flow(config_entry: ConfigEntry):
```

Comment:
Missing return type:

```suggestion
    def async_get_options_flow(config_entry: ConfigEntry) -> KeeneticOptionsFlowHandler:
```